include/dlaf/eigensolver/reduction_to_band/mc.h

```diff
@@ -270,28 +274,37 @@ void updateTrailingPanelWithReflector(const std::vector<TileT<T>>& panel, SizeTy
 }
 
 template <class T>
-pika::shared_future<common::internal::vector<T>> computePanelReflectors(
-    MatrixT<T>& mat_a, const common::IterableRange2D<SizeType, matrix::LocalTile_TAG> ai_panel_range,
-    SizeType nrefls) {
-  auto panel_task = pika::unwrapping([nrefls, cols = mat_a.blockSize().cols()](auto fut_panel_tiles) {
+pika::shared_future<common::internal::vector<T>> computePanelReflectors(MatrixT<T>& mat_a,
+                                                                        const SubPanelView& panel_view,
+                                                                        const SizeType nrefls) {
+  auto panel_task = pika::unwrapping([nrefls, width = panel_view.cols()](auto fut_panel_tiles) {
```

Comment:
I don't know what to think about `cols`/`width` now :D https://github.com/eth-cscs/DLA-Future/pull/456#discussion_r806628673. Was there a problem with calling the variable `cols`?

Reply:
Again #456...